<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meshBlendedShaderView</class>
 <widget class="QWidget" name="meshBlendedShaderView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>448</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>448</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>448</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="6" column="1">
    <widget class="QSpinBox" name="spinBox_tileFactor">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Tile factor</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSlider" name="horizontalSlider_tileFactor">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="TextureDrop" name="textureDrop_diffuseB" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>71</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(90, 90, 90);</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="TextureDrop" name="textureDrop_diffuseG" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>71</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(70, 70, 70);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="TextureDrop" name="textureDrop_diffuseR" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>71</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(90, 90, 90);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="TextureDrop" name="textureDrop_blendmap" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>71</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(70, 70, 70);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="TextureDrop" name="textureDrop_glow" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>71</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(90, 90, 90);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="TextureDrop" name="textureDrop_specular" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>71</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(70, 70, 70);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextureDrop</class>
   <extends>QWidget</extends>
   <header location="global">RootTools/Treenity/Include/TextureDrop.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_tileFactor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_tileFactor</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_tileFactor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_tileFactor</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
